tools/s3segmenter/s3segmenter.xml

```diff
@@ -0,0 +1,192 @@
+        <param name="stackProbPath" type="data" format="tiff" optional="true" label="Stack Probabilities"/>
+        <param name="probMapChan" type="integer" value="-1" label="Probability Maps Channel"/>
+
+        <conditional name="crop_select">
+            <param name="crop" type="select" label="Crop Strategy">
+                <option selected="true" value="noCrop">No Crop</option>
+                <option value="autoCrop">Automatic Crop</option>
+                <option value="dearray">De-array</option>
+                <option value="plate">Plate</option>
+            </param>
+            <when value="dearray">
+                <param name="maskPath" type="data" format="tiff" label="TMA Mask File"/>
+            </when>
+        </conditional>
+
+        <param name="cytoMethod" type="select" label="Cyto Method">
+            <option value="hybrid">Hybrid</option>
+            <option selected="true" value="distanceTransform">Distance Transform</option>
+            <option value="bwdistanceTransform">BW Distance Transform</option>
+            <option value="ring">Ring</option>
+        </param>
+        <param name="nucleiFilter" type="select" label="Nuclei Filter">
+            <option selected="true" value="IntPM">IntPM</option>
+            <option value="LoG">LoG</option>
+            <option value="Int">Int</option>
+            <option value="none">none</option>
+        </param>
+
+        <conditional name="nucleiRegion_select">
+            <param name="nucleiRegion" type="select" label="Nuclei Region">
+                <option value="watershedContourDist">watershedContourDist</option>
+                <option selected="true" value="watershedContourInt">watershedContourInt</option>
+                <option value="watershedBWDist">watershedBWDist</option>
+                <option value="dilation">dilation</option>
+                <option value="localThreshold">localThreshold</option>
+                <option value="localMax">localMax</option>
+                <option value="bypass">bypass</option>
+                <option value="pixellevel">pixellevel</option>
+            </param>
+            <when value="pixellevel">
+                <param name="pixelThreshold" type="float" value="-1.0" label="Pixel Threshold"/>
+                <param name="pixelMaskChan" type="text" value="2" label="Pixel Mask Channel"/>
+            </when>
+        </conditional>
+
+        <param name="segmentCytoplasm" type="select" label="Segment Cytoplasm">
+            <option value="segmentCytoplasm">segmentCytoplasm</option>
+            <option selected="true" value="ignoreCytoplasm">ignoreCytoplasm</option>
+        </param>
+        <param name="saveMask" type="boolean" checked="true" truevalue="--saveMask" falsevalue="" label="Save Mask"/>
+        <param name="saveFig" type="boolean" checked="true" truevalue="--saveFig" falsevalue="" label="Save Figure"/>
```

Comment:
we could use `argument="--saveFig"` here and remove name ... using argument is preferred if we can use it. It will also add something nice to the help text of the parameter :)